lib/puppet_x/elastic/plugin_parsing.rb

```diff
@@ -17,7 +17,7 @@ def self.plugin_version(raw_name)
     end
 
     # Attempt to guess at the plugin's final directory name
-    def self.plugin_split(original_string, position, soft_fail: true)
+    def self.plugin_split(original_string, position, soft_fail = true) # rubocop:disable Style/OptionalBooleanParameter
```

Comment:
This seems to be called only once on line 13. What about following rubocop advice and fix that line:

```diff
-      v = plugin_split(raw_name, 2, false).gsub(%r{^[^0-9]*}, '')
+      v = plugin_split(raw_name, 2, soft_fail: false).gsub(%r{^[^0-9]*}, '')
```